### PR TITLE
Add documentation for rest tagpage and operationpage plugins

### PR DESCRIPTION
**Pull request by hellosnow**

### CODE REVIEW

File: Documentation/templates-and-plugins/plugins-dashboard.md [hunk 1]
```diff
@@ -15,6 +15,30 @@ templates:
...
         command: "-t default,<output>/memberpage.<version>/content"
         config: 'template: ["statictoc", "<output>/memberpage.<version>/content"]'
+    - name: rest.tagpage
+      description: It splits the *REST* model into tag level model. With this plugin enabled, operations with the same tag are grouped into one page. If the operation is in multiple tags, it would be included in first tag level page.
+      type: Internal
+      thumbnail: ~/templates-and-plugins/images/rest.tagpage.default.screenshot.png
+      homepage: https://www.nuget.org/packages/rest.tagpage/
+      repository:
+        type: git
+        url: "https://github.com/dotnet/docfx/tree/master/plugins/Microsoft.DocAsCode.Build.TagLevelRestApi"
```
Comment (superyyrrzz) [inline Documentation/templates-and-plugins/plugins-dashboard.md]: >TagLevelRestApi [](start = 92, length = 15)

TagsLevelRestApi

File: Documentation/tutorial/intro_rest_api_documentation.md [hunk 2]
```diff
@@ -224,4 +224,13 @@ You can define your own metadata with YAML header. This functionality is quite u
...
 
-When the key of the metadata is not preserved by DocFX, for example, `not_predefined`. The metadata is kept and can be used in the template.
\ No newline at end of file
+When the key of the metadata is not preserved by DocFX, for example, `not_predefined`. The metadata is kept and can be used in the template.
+
+## Split extensibility
+By default, one *REST* API file generates one HTML file. For example, petstore.json generates petstore.html. We provide `rest.tagpage` and `rest.opearationpage` plugins to split the original *REST* API page into smaller pages.
+
+1. With `rest.tagpage` plugin enabled, operations with the same tag are grouped into one page.
+2. With `rest.operationpage` plugin enabled, each operation is spliited into single page.
```
Comment (superyyrrzz) [inline Documentation/tutorial/intro_rest_api_documentation.md]: >spliited [](start = 63, length = 8)

splitted

**Comment by superyyrrzz** [comment]
:shipit:

File: Documentation/tutorial/intro_rest_api_documentation.md [hunk 3]
```diff
@@ -224,4 +224,13 @@ You can define your own metadata with YAML header. This functionality is quite u
 
 When the key of the metadata is already preserved by DocFX, for example, `summary`, the value of `summary` will be overwritten. You can also overwrite complex types, for example, `description` of a `parameter`. Make sure the data structure of the provided metadata is consistent with the one defined in DocFX, otherwise, DocFX is unable to cast the value and fails.
 
-When the key of the metadata is not preserved by DocFX, for example, `not_predefined`. The metadata is kept and can be used in the template.
\ No newline at end of file
+When the key of the metadata is not preserved by DocFX, for example, `not_predefined`. The metadata is kept and can be used in the template.
+
+## Split extensibility
+By default, one *REST* API file generates one HTML file. For example, petstore.json generates petstore.html. We provide `rest.tagpage` and `rest.opearationpage` plugins to split the original *REST* API page into smaller pages.
```
Comment (928PJY) [inline Documentation/tutorial/intro_rest_api_documentation.md]: >opearationpage [](start = 145, length = 14)

operationpage?

**Comment by 928PJY** [comment]
:shipit:

**Pull request merged**